### ISSUE TITLE
Highlight all is not highlighing the correct results when there are multiple matches in the same line.

### ISSUE THREAD

**Issue by Hypercubed** [stephanrauh/ngx-extended-pdf-viewer#2356]
**Describe the bug**
Highlight all is not highlighing the correct results when there are multiple matches in the same line.

**Version info**
20.2.0

**Demo PDF file**
Can be seen in the demo (https://pdfviewer.net/extended-pdf-viewer/simple) when searching for "the":

<img width="1182" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/509946/1ae8fec2-ab06-4f8b-82fc-840c48ab850a">

By comparing to pdfjs demo I think the folllowing CSS is missing:

```
  .textLayer .highlight.appended {
    position: initial;
  }
```

Adding that manually in chrome devtools seams to fix it:

<img width="1181" alt="image" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/509946/10813194-552d-4a2a-b645-e17f3c2cbff8">

**Comment by stephanrauh** [comment]
Yes, you're right. It was a single closing curly brace at the wrong position. Thanks for pointing that out, and special kudos for providing the missing CSS rule. It allowed me to find the bug a lot faster!

**Comment by stephanrauh** [comment]
Your bugfix has landed with version 20.5.0-alpha.2.

Enjoy!
Stephan
